### DISCUSSION TITLE
Quick switch between projects without exit lazygit

### DISCUSSION THREAD

**Discussion by LintaoAmons** [jesseduffield/lazygit#1886] [Ideas]
It will be very nice to have a shortcut to toggle a pop-up window to select the available git project.

**Comment by nagy135** [comment]
isnt that already possible? i mean `1 Enter` gets you to repos you opened already and you can even search those.

**Reply by LintaoAmons** [reply]
I'm sorry for my dummy question~
I didn't notice that the `status pane` can expand to select the projects.

Thanks for your hint~
